warehouse/accounts/models.py

```diff
@@ -31,16 +34,25 @@
     select,
     sql,
 )
-from sqlalchemy.dialects.postgresql import CITEXT, UUID
+from sqlalchemy.dialects.postgresql import CITEXT, UUID as PG_UUID
 from sqlalchemy.exc import NoResultFound
 from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.orm import Mapped, mapped_column
 
 from warehouse import db
 from warehouse.events.models import HasEvents
 from warehouse.sitemap.models import SitemapMixin
 from warehouse.utils.attrs import make_repr
 from warehouse.utils.db.types import TZDateTime
 
+if TYPE_CHECKING:
+    from warehouse.macaroons.models import Macaroon
+    from warehouse.oidc.models import PendingOIDCPublisher
+
+
+# Custom column types
+bool_false = Annotated[bool, mapped_column(Boolean, server_default=sql.false())]
```

Comment:
I mentioned that in commit message on 269c94bb8cd92e60df3b7e81a37ae9ad66c98718 
If this is more widely useful, I'm thinking `warehouse/utils/db/types.py` since we already have that module.